### ISSUE TITLE
`prost_path` not updating macro calls to the prost crate

### ISSUE THREAD

**Issue by liyongjing** [tokio-rs/prost#806]
1.  missing replace the prost_path?

::prost::alloc::boxed::Box<{}>
::prost::alloc::collections::BTreeMap
::prost::alloc::vec::Vec
::prost::bytes::Bytes

repeated bytes test2 = 2;
pub test2: ::my_crate::prost::alloc::vec::Vec<::prost::alloc::vec::Vec<u8>>,
::prost::alloc::vec::Vec<u8> miss

2. ::my_crate::prost::Message throw error  with other crate.  similar using https://serde.rs/container-attrs.html#crate ?

bug demo :
[prost_path_test.zip](https://github.com/tokio-rs/prost/files/10575757/prost_path_test.zip)

> 1. generate proto rust file at domain/proto.rs
> cargo r --example proto
> 2. run test
> cargo r my_test

**Comment by LucioFranco** [comment]
Could you explain more what the bug is?

**Comment by LucioFranco** [comment]
Yeah, I think the prost_path might be a bit bugged for derive. Its using absolute paths here https://github.com/tokio-rs/prost/blob/master/prost-derive/src/lib.rs#L440 which I think is incorrect, though I am not sure how to feed the correct info into the macro system.
